src/main/java/com/github/splendor_mobile_game/websocket/handlers/Message.java

```diff
@@ -1,9 +1,19 @@
 package com.github.splendor_mobile_game.websocket.handlers;
 
+/** Represents a message to be sent to a receiver identified by their websocket hashcode. */
 public class Message {
+
+    /** The websocket hashcode of the receiver of the message. */
```

Comment:
"Each connection between User and Server is identified by some ID. `receiverHashcode` is the ID of the connection.